beancount_dkb/ec.py

```diff
@@ -46,6 +46,10 @@ def __init__(self, iban, account, currency='EUR',
     def file_account(self, _):
         return self.account
 
+    def file_date(self, file_):
+        self.extract(file_)
```

Comment:
Just as an FYI, I might later on extract a `_parse_metadata()` function from the current implementation of `extract()`, which could then be used here instead of parsing the entire file.